ext/standard/string.c

```diff
@@ -1362,29 +1362,33 @@ PHPAPI zend_string *php_string_toupper(zend_string *s)
 	unsigned char *c;
 	const unsigned char *e;
 
-	c = (unsigned char *)ZSTR_VAL(s);
-	e = c + ZSTR_LEN(s);
+	if (EXPECTED(!BG(ctype_string))) {
+		return zend_string_toupper(s);
+	} else {
```

Comment:
The else + reindent is not necessary. Fixed for tolower in https://github.com/php/php-src/commit/1441271f18bcadcaf4c192654b06e70d475dfec9.

Reply:
Done in the force push